daemon/info.go

```diff
@@ -111,3 +112,13 @@ func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 
 	return v, nil
 }
+
+// The uppercase and the lowercase are available for the proxy settings.
+// See the Go specification for details on these variables. https://golang.org/pkg/net/http/
+func getProxyEnv(key string) string {
+	proxyValue := os.Getenv(strings.ToUpper(key))
+	if proxyValue == "" {
+		return os.Getenv(key)
```

Comment:
@phemmer I mentioned it in docs/reference/commandline/cli.md
https://github.com/nak3/docker/commit/8b1fc7f59957473a0971bfcf93226896378a6317